scala-libraries-2/src/test/scala/com/baeldung/pureconfig/SampleConfigLoader.scala

```diff
@@ -0,0 +1,57 @@
+package com.baeldung.pureconfig
+
+import com.typesafe.config.ConfigFactory.parseString
+import pureconfig.ConfigSource
+import pureconfig._
+import pureconfig.generic.auto._
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatest.matchers.should.Matchers
+import impl._
+import pureconfig.module.enumeratum._
+import pureconfig.generic.ProductHint
+import pureconfig.error.ConfigReaderException
+
+class SampleConfigLoader extends AnyWordSpec with Matchers {
+
+  "pureconfig" should {
+    "load config successfully" in {
+      val httpConf = ConfigSource.default.at("http").load[HttpConfig]
+      val monitoringConf =
+        ConfigSource.default.at("monitoring").load[MonitoringConf]
+
```

Comment:
Let's remove this blank space and the blank space on line 24.

Reply:
Done